arkouda-helm-charts/arkouda-udp-server/templates/service-rolebinding.yaml

```
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: arkouda-service-crud
+subjects:
+- kind: User
+  name: arkouda
```

should the `name` be `{{ .Values.user }}` like in the other rolebinding yamls?
```suggestion
- kind: User
  name: {{ .Values.user }}
```

---

@hokiegeek2 should this name be updated to match the other yamls?

---

@pierce314159 wow, good catch! checking

---

@pierce314159 yup, excellent catch, fixed!